R/s3.R

```diff
@@ -104,7 +104,8 @@ s3_put <- function(
   aws.signature::use_credentials(profile = s3_config$profile)
   
   # determine whether and where the remote file exists
-  bucket_contents <- aws.s3::get_bucket_df(bucket = s3_config$bucket)
+  bucket_contents <- aws.s3::get_bucket_df(bucket = s3_config$bucket,
+                                           prefix = dirname(data_file))
```

Comment:
My only comment would be to add either comments, warnings, or that `max=Inf` now in order to save us future headaches if this pops up again.

One idea: if the bucket contents that comes back hits that 1000 item limit, we could send the user a warning to check that all their contents made it.
```suggestion
                                           prefix = dirname(data_file))
if(length(bucket_contents) >= 1000) warning('The S3 bucket returned items around the max limit (1000). Please verify that all your expected items were returned')
```

Reply:
Good idea, I will add something similar

Reply:
added max = Inf here